stdlib/public/SDK/Dispatch/Time.swift

```diff
@@ -13,6 +13,12 @@
 import _SwiftDispatchOverlayShims
 
 public struct DispatchTime : Comparable {
+	private static let timebaseInfo: mach_timebase_info_data_t = {
```

Comment:
I believe it only breaks SIL printing and -sil-serialize-all these days, so it should be okay. Foundation's using it plenty and some of the tiny overlays already use it too.